internal/controller/management_controller.go

```diff
@@ -442,10 +476,13 @@ func (r *ManagementReconciler) ensureAccessManagement(ctx context.Context, mgmt
 	}
 
 	if err := r.Client.Create(ctx, amObj); err != nil {
-		return fmt.Errorf("failed to create %s AccessManagement object: %w", kcm.AccessManagementName, err)
+		msg := fmt.Sprintf("failed to create %s AccessManagement object: %v", kcm.AccessManagementName, err)
+		record.Warn(mgmt, nil, "AccessManagementCreateFailed", msg)
+		return errors.New(msg)
```

Comment:
```suggestion
		err = fmt.Sprintf("failed to create %s AccessManagement object: %w", kcm.AccessManagementName, err)
		record.Warn(mgmt, nil, "AccessManagementCreateFailed", msg.Error())
		return err
```

To avoid losing the original error